src/storage/src/keyspace.rs

```diff
@@ -30,7 +30,7 @@ pub struct Keyspace<S: StateStore> {
     store: S,
 
     /// Encoded representation for all segments.
-    prefix: Vec<u8>,
+    pub prefix: Vec<u8>,
```

Comment:
Probably add a method instead of making it public

Reply:
Sounds good!

Reply:
> I still think the state table should be built based on Keyspace, and none of the interfaces should be aware of the keyspace's prefix. This will definitely make the implementation much clearer and bug-freer. 🥹

I'd like to do this refactor while reviewing the state table these days.